### DISCUSSION TITLE
Will this run on .NET Framework 4.7.2?

### DISCUSSION THREAD

**Discussion by wheelersf** [abuzuhri/Amazon-SP-API-CSharp#30] [Q&A]
Greetings,

Very nice library! Well done!

Will this work with .NET Framework 4.7.2 currently? Or, do I need to update to .NET 5 to support the dependencies?

Thanks!

**Comment by abuzuhri** [comment]
Its will run to any .Net version after v  4.7.1 , since it build with .Net standard
